src/makielayout/blocks/axis.jl

```diff
@@ -919,9 +919,12 @@ end
 """
     autolimits!(la::Axis)
 
-Reset manually specified limits of `la` to an automatically determined rectangle, that depends on the data limits of all plot objects in the axis, as well as the autolimit margins for x and y axis.
+Reset manually specified limits of `la` to an automatically determined rectangle, that depends on the 
+data limits of all plot objects in the axis, as well as the autolimit margins for x and y axis.
+
+If no axis is provided, `la` defaults to `current_axis()`.
 """
-function autolimits!(ax::Axis)
+function autolimits!(ax::Axis=current_axis())
```

Comment:
I'm not sure if this makes sense, because `current_axis()` could potentially return an axis object that doesn't have the `autolimits!` function implemented. The definition of what can be `current_axis()` is very loose, it's just whatever defines `Makie.can_be_current_axis` to be `true`